modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -141,13 +113,17 @@ const EditorUpload = (editor: Editor): EditorUpload => {
 
       return uploader.upload(blobInfos, openNotification(editor)).then(aliveGuard((result) => {
         const imagesToRemove: HTMLImageElement[] = [];
+        let shouldDispatchChange = false;
 
         const filteredResult: UploadResult[] = Arr.map(result, (uploadInfo, index) => {
           const blobInfo = imageInfos[index].blobInfo;
           const image = imageInfos[index].image;
           let removed = false;
 
           if (uploadInfo.status && Options.shouldReplaceBlobUris(editor)) {
+            if (uploadInfo.url && !image.src.includes(uploadInfo.url)) {
```

Comment:
Please use the Katamari methods instead here, so this should be `!Strings.contains(image.src, uploadInfo.url)`.

Reply:
changed